InitNiKom/InitNiKom.c

```diff
@@ -28,7 +28,7 @@ void createDirOrDie(char *dirname) {
   BPTR lock;
   struct FileInfoBlock __aligned fib;
   
-  if(lock = Lock(dirname, ACCESS_READ)) {
+  if((lock = Lock(dirname, ACCESS_READ))) {
```

Comment:
So the rule in gcc is "if there is an extra set of parens it won't generate a warning if there is an assignment in the if condition"?
The problem with this is that there is no such trick in SAS/C so anyone developing with SAS/C is at risk of introducing warnings in gcc without realizing it. But I guess it's not a huge problem..